docs/guides/examples/aggregate_function_combinators/sumForEach.md

```diff
@@ -0,0 +1,48 @@
+---
+slug: '/examples/aggregate-function-combinators/sumForEach'
+title: 'sumForEach'
+description: 'Example of using the sumArray combinator'
+keywords: ['sum', 'array', 'combinator', 'examples', 'sumArray']
+sidebar_label: 'sumArray'
+---
+
+# sumArray {#sumforeach}
+
+## Description {#description}
+
+The [`ForEach`](/sql-reference/aggregate-functions/combinators#-foreach) combinator
+can be applied to the [`sum`](/sql-reference/aggregate-functions/reference/sum) aggregate function to turn it from an aggregate
+function which operates on tables to an aggregate function which operates on 
```

Comment:
This doesnt sound quite right. The sum function operates on row values..but the ForEach operates on array columns applying the aggregate to each  element in the array across rows